src/Types.ts

```diff
@@ -22,7 +22,13 @@ type file = {
 
 // Contains information necessary to make a track
 type track = {
-  files: Array<file>
+  trackFile: file;
+  trackType: filetype;
+  trackColorSettings: ColorScheme;
+}
```

Comment:
Now we have duplicate copies of the type for the file.

We should probably get rid of `file`, and just have `track` have a filename string and a track type.

Or, we could store the type only in the file and not in the track.

But we should not store it twice.